CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# dbt_twilio_source v0.3.0
+
+## 🚨 Breaking Changes 🚨
+- Added explicit float casts to the following numerical fields: `duration, queue_time, num_media, num_segments, count` in addition to the existing `duration, price, count, usage` fields. These fields are listed as strings by default in the Twilio API, but would only be valuable as numbers. Please be aware if you were expecting string values from these fields.
```

Comment:
`duration` and `count` are included in both segments. Looking at the code, it looks as if they only belong to the first segment, so I'd remove them from the second section.

Reply:
oh sharp eye!! Thank you, updated